core/primitives-core/src/config.rs

```diff
@@ -318,14 +318,14 @@ impl ExtCostsConfig {
             ExtCosts::storage_remove_ret_value_byte => SAFETY_MULTIPLIER * 3843852,
             ExtCosts::storage_has_key_base => SAFETY_MULTIPLIER * 18013298875,
             ExtCosts::storage_has_key_byte => SAFETY_MULTIPLIER * 10263615,
-            ExtCosts::storage_iter_create_prefix_base => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_create_prefix_byte => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_create_range_base => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_create_from_byte => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_create_to_byte => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_next_base => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_next_key_byte => SAFETY_MULTIPLIER * 0,
-            ExtCosts::storage_iter_next_value_byte => SAFETY_MULTIPLIER * 0,
+            ExtCosts::storage_iter_create_prefix_base => 0,
```

Comment:
Should we leave a small comment to explain why these values do not have the `SAFETY_MULTIPLIER *` prefix?